controllers/infra/config/samples/infra_v1_infraresource.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: infra.sealos.io/v1
+kind: InfraResource
+metadata:
+  name: infraresource-sample
+  namespace: metering-system
+spec:
+  resourceName: infra
+  interval: 60
```

Comment:
you should interval is 1 Minute?
And the price should change it

Reply:
> you should interval is 1 Minute? And the price should change it

I've changed the cost calculation to divide 60